src/main/java/com/wavefront/AbstractTraceLoader.java

```diff
@@ -61,6 +65,8 @@ public void start(String[] args) {
     } catch (Throwable t) {
       handleProgramExit(t);
     }
+    System.out.println("\u001B[33m" + String.format("Max heap memory: %.2f GB", maxHeapMemory) );
+    System.out.println("\u001B[33m" + String.format("Used heap memory: %.2f GB", usedHeapMemory));
```

Comment:
I prefer to see something like `ANSI_YELLOW` instead of `\u001B[33m`.
It is defined [here](https://gist.github.com/mgumiero9/665ab5f0e5e7e46cb049c1544a00e29f)